### PR TITLE
SAK-32113 Wrong order when using the 'Download'option in STUDENT MEMB…

### PR DESCRIPTION
**Pull request by manuelal**
…ERSHIPS from Section Info

### CODE REVIEW

File: sections/sections-app/src/java/org/sakaiproject/tool/section/jsf/backingbean/RosterBean.java [hunk 1]
```diff
@@ -338,17 +339,16 @@ public void export(ActionEvent event){
         	
         } 
         spreadsheetData.add(header);
-        for(Iterator enrollmentIter = siteStudents.iterator(); enrollmentIter.hasNext();) {
-            //EnrollmentDecorator enrollment = enrollmentIter.next();
-        	ParticipationRecord record = (ParticipationRecord)enrollmentIter.next();
+        for(Iterator enrollmentIter = unpagedEnrollments.iterator(); enrollmentIter.hasNext();) {
```
Comment (ottenhoff) [inline sections/sections-app/src/java/org/sakaiproject/tool/section/jsf/backingbean/RosterBean.java]: Do you need to use an iterator here? 

`for (EnrollmentDecorator enrollment  : unpagedEnrollments ) {`

File: sections/sections-app/src/java/org/sakaiproject/tool/section/jsf/backingbean/RosterBean.java [hunk 2]
```diff
@@ -76,6 +76,7 @@
 	private List<EnrollmentDecorator> enrollments;
 	private List<String> categories;
     private List<EnrollmentRecord> siteStudents;
+    private List<EnrollmentDecorator> unpagedEnrollments;
```
Comment (ottenhoff) [inline sections/sections-app/src/java/org/sakaiproject/tool/section/jsf/backingbean/RosterBean.java]: Can you fix the tabs/spaces here with siteStudents and unpagedEnrollments?

**Comment by ottenhoff** [comment]
Has this been tested?

**Comment by jmarianolujan** [comment]
Hi Sam,

Its been tested locally and in a nightly server, actually, the patch is applied in that server and can be tested right now.  - https://sakai-trunk.atica.um.es/portal  (see applied patches)

Cheers

**Pull request merged**